transport/zipkin/http.go

```diff
@@ -1,6 +1,6 @@
-// Copyright (c) 2016 The OpenTracing Authors
-// Copyright (c) 2016 Bas van Beek
-// Copyright (c) 2016 Uber Technologies, Inc.
+// Copyright (c) 2017 The OpenTracing Authors
+// Copyright (c) 2017 Bas van Beek
```

Comment:
no, it's not. We should keep the original notices. Why is this changing from Uber to OT Authors?